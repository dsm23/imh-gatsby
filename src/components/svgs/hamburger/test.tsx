/// <reference types="jest" />

import React from "react";
import { render } from "@testing-library/react";

import Hamburger from ".";

const renderable = <Hamburger />;

describe(`Cross`, () => {
  it(`it renders`, () => {
    const { baseElement } = render(renderable);

    expect(baseElement).toBeTruthy();
  });

  it(`matches snapshot`, () => {
    const { asFragment } = render(renderable);

    expect(asFragment()).toMatchSnapshot();
  });
});
