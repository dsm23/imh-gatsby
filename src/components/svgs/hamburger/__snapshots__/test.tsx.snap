// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cross matches snapshot 1`] = `
<DocumentFragment>
  <svg
    fill="#f4f5f7"
    stroke="currentColor"
    viewBox="0 0 24 24"
  >
    <path
      d="M4 6h16M4 12h16M4 18h16"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="2"
    />
  </svg>
</DocumentFragment>
`;
